fix(css): update path() and shape-outside pages (#34961)
mdn/content: files/en-us/web/css/shape-outside/index.md

      - : Defines the shape enclosed by the outside content edge. Each corner radius of this box is the larger of `0` or `border-radius - border-width - padding`.

- {{cssxref("&lt;basic-shape&gt;")}}

>   - : The float area is computed based on the shape created by one of the {{cssxref("basic-shape/inset","inset()")}}, {{cssxref("basic-shape/circle","circle()")}}, {{cssxref("basic-shape/ellipse","ellipse()")}}, and {{cssxref("basic-shape/polygon","polygon()")}} function, and other `<basic-shape>` functions are not allowed. If a `<shape-box>` is also supplied, it defines the reference box for the `<basic-shape>` function. Otherwise, the reference box defaults to `margin-box`.

Reviewer: ```suggestion
  - : The float area is computed based on the shape created by an {{cssxref("basic-shape/inset","inset()")}}, {{cssxref("basic-shape/circle","circle()")}}, {{cssxref("basic-shape/ellipse","ellipse()")}}, or {{cssxref("basic-shape/polygon","polygon()")}} function; other `<basic-shape>` functions are invalid. If a `<shape-box>` is also supplied, it defines the reference box for the `<basic-shape>` function. Otherwise, the reference box defaults to `margin-box`.
```

Reviewer: Also, I had a query about the sentence "If a `<shape-box>` is also supplied, it defines the reference box for the `<basic-shape>` function. Otherwise, the reference box defaults to `margin-box`." So, a possible value for the property can be a `<shape-box>` AND a `<basic-shape>`. No examples of this are shown anywhere above or in the examples section. Shouldn't you at least include a couple of syntax examples?

Reviewer: @OnkarRuikar I'm assuming this last comment here is the only one you've not yet actioned. Let me know when you've decided what to do with it. Thanks!

Author: I was working on other stuff, so I kept this comment open. I've added two examples of shape box with basic shape in the syntax section above.
